src/Date/Local.elm

```diff
@@ -0,0 +1,111 @@
+module Date.Local exposing (..)
+
+{-| Record to store localized time format.
+
+Time zones and default formats are not implemented,
+but included to avoid possible version conflicts in the future.
+-}
+
+type alias Local =
+  { date :
+    { months : Months
+    , monthsAbbrev : Months
+    , wdays : WDays
+    , wdaysAbbrev : WDays
+    , defaultFormat : Maybe String
```

Comment:
I added the defaultFormats for the locale-dependent date and time formats `%c`, `%x`, and `%X`. Added comments in that regard.